src/error.rs

```diff
@@ -154,7 +154,7 @@ pub(crate) enum Error<'src> {
     recipe: &'src str,
   },
   Signal {
-    recipe: &'src str,
+    recipe: Option<&'src str>,
```

Comment:
it's possible for the user to just do `just --chooser` without specifying the recipe, so I changed this into an `Option`